Makefile

```diff
@@ -1,86 +1,24 @@
 SHELL = /bin/bash
 GO ?= go
 CC ?= gcc
-COVERAGE_PATH ?= $(shell pwd)/.coverage
-CRIU_FEATURE_MEM_TRACK = $(shell if criu check --feature mem_dirty_track > /dev/null; then echo 1; else echo 0; fi)
-CRIU_FEATURE_LAZY_PAGES = $(shell if criu check --feature uffd-noncoop > /dev/null; then echo 1; else echo 0; fi)
-CRIU_FEATURE_PIDFD_STORE = $(shell if criu check --feature pidfd_store > /dev/null; then echo 1; else echo 0; fi)
 
-export CRIU_FEATURE_MEM_TRACK CRIU_FEATURE_LAZY_PAGES CRIU_FEATURE_PIDFD_STORE
-
-all: build test phaul-test
+all: build
```

Comment:
Would we want to retain `test` here? Even though the test targets are moved into `test/Makefile`, we still have a `test` target to call it from this file.

Reply:
> Would we want to retain test here?

Not really. This change allows to run `make` (without sudo) to execute only the `build` target.
We can use `sudo make test` to run tests as root.